mobile-widgets/qml/Export.qml

```diff
@@ -12,6 +12,20 @@ Kirigami.ScrollablePage {
 
 	property int selectedExport: ExportType.EX_DIVE_XML
 
+	FileDialog {
```

Comment:
this I can see as its own commit (just continuing on the comments about not starting out with a "dummy" page

Reply:
no more "dummy" page.